deployments/container/Dockerfile.ubi9

```
@@ -38,13 +38,15 @@ ARG VERSION="N/A"
 ARG GIT_COMMIT="unknown"
 RUN make PREFIX=/artifacts cmds
 
+FROM nvcr.io/nvidia/cloud-native/k8s-mig-manager:v0.12.1-ubi9 as mig-manager
 
 FROM nvcr.io/nvidia/distroless/go:v3.1.8
```

Let's use the `-dev` tag instead which includes `busybox`. We need this now that the `chroot` binary may be invoked in `reconfigure_mig.go`
```suggestion
FROM nvcr.io/nvidia/distroless/go:v3.1.8-dev
```